neurobooth_os/iout/split_xdf.py

```diff
@@ -219,6 +219,7 @@ def create_h5_from_csv(dont_split_xdf_fpath, conn, server_name=None):
         for row in lines:
             # change to NAS path if necessary
             if not os.path.exists(row[0]):
+                row[0] = row[0].replace('\\', '/')
```

Comment:
I'm having trouble seeing the big picture here. Is this string related to the remote_data_dir from the config?  I'm just wondering if there is no way to write it consistently in both places so this rewrite isn't needed.

Reply:
The configs have strings with forward slashes. The string in the csv file are generated by the get_xdf_name function in split_xdf.py and contain backslashes. Ideally the configs should have backslashes (since that is the Windows default) and we wouldn't need to do the replace.

Reply:
Ah, that solves a mystery then, I always wondered why we needed 4 backslashes!!